.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: monthly
```

Comment:
Since we're deploying more frequently than monthly, can we make this interval weekly?

Reply:
Will make this change in a separate PR.